scripts/2.mutation-classifier.py

```diff
@@ -48,46 +70,76 @@
 
 # In[4]:
 
-get_ipython().run_cell_magic('time', '', "path = os.path.join('download', 'expression-matrix.tsv.bz2')\nexpression_df = pd.read_table(path, index_col=0)")
+
+path = os.path.join('download', 'expression-matrix.tsv.bz2')
+expression_df = pd.read_table(path, index_col=0)
+
+path = os.path.join('download', 'mutation-matrix.tsv.bz2')
+mutation_df = pd.read_table(path, index_col=0)
+
+path = os.path.join('download', 'covariates.tsv')
+covariate_df = pd.read_table(path, index_col=0)
 
 
 # In[5]:
 
-get_ipython().run_cell_magic('time', '', "path = os.path.join('download', 'mutation-matrix.tsv.bz2')\nmutation_df = pd.read_table(path, index_col=0)")
+
+# Select acronym_x and n_mutations_log1p covariates only
+disease_cols = [col for col in covariate_df.columns if col.startswith('acronym_')]
+
+# Filter covariate columns by disease if a list was provided
+if disease_acronyms:
+    disease_cols = [col for col in disease_cols if col.endswith(tuple(disease_acronyms))]
```

Comment:
Woah! Didn't know that in [`str.endswith`](https://docs.python.org/3/library/stdtypes.html), "suffix can also be a tuple of suffixes to look for". That is super convenient!